### ISSUE TITLE
build front end 1995 CH 5 - Personal Information

### ISSUE THREAD

**Issue by leannammiller** [department-of-veterans-affairs/vets-website#4546]
# 5.1 Contact Information
- How would you prefer to be contacted if VA has questions about your application? **[radio]**
- *Address **[standard address component]**
- *Email address **[email]**
- *Re-enter email address **[email]**
- Primary telephone number **[tel]** *_Required if phone is selected as preferred contact method_
- Mobile telephone number **[tel]**

# 5.2 Dependents
_Conditional based on 1978 question as asked in Military Service_

 - Are you currently married? **[radio]**
 - Do you have any children who are under age 18? Or do you have any children who are over age 18 but under 23, not married, and attending school? Or do you have any children of any age who are permanently disabled for mental or physical reasons? **[radio]**
 - Do you have a parent who is dependent on your financial support? **[radio]**

# 5.3 Direct Deposit
 - Do you want to start, stop or continue using direct deposit? **[radio]**
 - Account type **[radio]**
 - Account number **[number]**
 - Routing number **[number]**

**Comment by akainic** [comment]
@cweber @goldenmeanie are these all on the same page? And what exactly is the dependents question? Number of dependents?

**Comment by cweber** [comment]
Updated. I think the dependents thing was a page title.

**Comment by cweber** [comment]
@mollyblake This is the last chapter for the 1995, but I don't think we have any copy needs for it. It's all just straight dupes of fields from the 1990. Just pining you for awareness.

**Comment by cweber** [comment]
Required fields indicated by an *

**Comment by mollyblake** [comment]
@cweber agree.  The only thing I might change is "How would you prefer to be contacted if VA has questions about your application? [radio]"  to "How would you LIKE to be contacted ..."  

Thanks!

**Comment by annekainicUSDS** [comment]
Notes from meeting with Shay:

- Indicating "Phone" for preferred contact method should make "Phone" fields required

**Comment by jbalboni** [comment]
Above is fixed in #4789

**Comment by goldenmeanie** [comment]
From @mollyblake 
```
Do you have any children who fall into these categories?
- Under the age of 18
- Between the ages of 18 and 22, not married, and attending school
- Any age who are permanently disabled for mental or physical reasons
```
